thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/resolution/component.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Filter Bar Component
+ * Constructs a filter bar based on a config file
+ * @module custom/anomalies-table/resolution
+ * @property {object[]} config            - [required] array of objects (config file) passed in from the route that sets
+ *                                          up the filter bar sub-filters
+ * @property {object} entities            - [required] data used to filter
+ * @property {array}  onFilterSelection   - [required] closure action to bubble to controller on filter selection
+ *
+ * @example for usage in models table columns definitions
+ *   {
+ *     propertyName: 'feedback',
+ *     component: 'custom/anomalies-table/resolution',
+ *     title: 'Resolution',
+ *     className: 'anomalies-table__column',
+ *     disableFiltering: true
+ *   },
+ */
+import Component from "@ember/component";
+import {
+  anomalyResponseObj,
+  getAnomalyDataUrl,
+  updateAnomalyFeedback,
+  verifyAnomalyFeedback
+ } from 'thirdeye-frontend/utils/anomaly';
+import { set } from '@ember/object';
+
+export default Component.extend({
+  tagName: '',//using tagless so i can add my own in hbs
+  anomalyResponseNames: anomalyResponseObj.map(response => response.name),
+  anomalyDataUrl: getAnomalyDataUrl(),
+
+  actions: {
+    /**
+     * Handle dynamically saving anomaly feedback responses
+     * @method onChangeAnomalyResponse
+     * @param {Object} anomalyRecord - the anomaly being responded to
+     * @param {String} selectedResponse - user-selected anomaly feedback option
+     * @param {Object} inputObj - the selection object
+     */
+    onChangeAnomalyResponse(anomalyRecord, selectedResponse, inputObj) {
+      const responseObj = anomalyResponseObj.find(res => res.name === selectedResponse);
+      set(inputObj, 'selected', selectedResponse);
+
+      // Save anomaly feedback
+      updateAnomalyFeedback(anomalyRecord.anomalyId, responseObj.value)
+        .then((res) => {
+          // We make a call to ensure our new response got saved
+          verifyAnomalyFeedback(anomalyRecord.anomalyId, responseObj.status)
+            .then((res) => {
+              const filterMap = res.searchFilters ? res.searchFilters.statusFilterMap : null;
+              if (filterMap && filterMap.hasOwnProperty(responseObj.status)) {
+                set(anomalyRecord, 'anomalyFeedback', selectedResponse);
```

Comment:
if `anomalyRecord` is an ember object (it should be), 

we can utilize `setProperties` so that we don't notify property changes for each sets. An alternative would be to wrap that in an ember run. 

### documentation: 
https://emberjs.com/api/ember/2.18/functions/@ember%2Fobject/setProperties

### example:
```
anomalyRecord.setProperties({
  anomalyFeedback: selectedResponse,
  showResponseSaved: true
})
```

Reply:
again existing codes. But updated. Thanks.

Reply:
DONE